death-clock.css

```diff
@@ -34,6 +37,12 @@
   margin-top: -2px;
 }
 
+#death-clock-youtube-dark {
+  color:white;
```

Comment:
Can we add a blank space before `white`?

Reply:
We have some duplicated CSS here. Instead of making a new ID, let's just use a `.dark` class. I added more details in a comment below!